lib/services/common.dart

```diff
@@ -4,9 +4,13 @@
 
 library dart_pad.common;
 
-final String serverURL = 'https://dart-services.appspot.com/';
-//final String serverURL = 'http://doc-test.dart-services.appspot.com/';
-//final String serverURL = 'http://127.0.0.1:8082/';
+
+// TODO: This is just for testing this PR remove when the dart-services are
+// updated and this PR is ready for merge
+
+//final String serverURL = 'https://dart-services.appspot.com/';
+final String serverURL = 'http://doc-test.dart-services.appspot.com/';
```

Comment:
Let's remove this, coordinate with @lukechurch re: the server, and plan to land this PR soon.